web/templates/topic_summary.hbs

```diff
@@ -1 +1 @@
-<p>{{message}}</p>
+<p>{{{message}}}</p>
```

Comment:
This should use the `{{rendered_markdown` helper, which is important for security.

Reply:
Okay, updated! I should have looked similar code here.